src/dev/pages/popover/popover.html

```diff
@@ -4,6 +4,7 @@
     title: 'Popover',
     includePath: './pages/popover/popover.ejs',
     options: [
+      { type: 'text-field', inputType: 'number', id: 'opt-delay', label: 'Delay', defaultValue: 0 },
```

Comment:
Let's rename the `label` to "Hover delay", and `id` to "opt-hover-delay" for consistency, and update the demo `.ts` file as well.